SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -48,52 +48,6 @@ DECLARE_CYCLE_STAT(TEXT("Sender QueueOutgoingUpdate"), STAT_SpatialSenderQueueOu
 DECLARE_CYCLE_STAT(TEXT("Sender UpdateInterestComponent"), STAT_SpatialSenderUpdateInterestComponent, STATGROUP_SpatialNet);
 DECLARE_CYCLE_STAT(TEXT("Sender SendRPC"), STAT_SpatialSenderSendRPC, STATGROUP_SpatialNet);
 
-namespace
```

Comment:
Instead of going through this and ActorSystem.cpp, can you call out any changes that aren't direct copies?

Reply:
had a scan- the only places that weren't direct copies were the stuff in spatial net driver with not updating SCV any more, and the logic around queuing updates until auth, so nothing left in actor system and sender

Reply:
Gave it a scan - looks good